### PR TITLE
Master offlinebaselines pr

### PR DESCRIPTION
**Pull request by alspellm**
Cleaning up and re-organizing some scripts. Added year option to Module Mapper

### CODE REVIEW

File: analysis/include/ModuleMapper.h [hunk 1]
```diff
@@ -10,167 +10,71 @@
...
-         * @brief description
-         * 
-         * @param filename 
-         */
-        void ReadThresholdsFile(std::string filename);
-
-        /**
-         * @brief description
-         * 
-         * @param feb
```
Comment (cbravo135) [inline analysis/include/ModuleMapper.h:138]: You are going to need to merge with master to make sure you don't lose all Sarah's recent work in adding these comments for Doxygen.

**Comment by alspellm** [comment]
Let me know if this PR fixed the doxygen problem

File: analysis/include/BlFitHistos.h [hunk 2]
```diff
@@ -22,98 +22,36 @@
 #include "TF1.h"
 #include <fstream>
 
-/**
```
Comment (cbravo135) [inline analysis/include/BlFitHistos.h:25]: This file is still losing the Doxygen comments Sarah added. Others are too, need to figure out how to update this PR so it doesn't delete her work.

**Review by cbravo135 — APPROVED** [review]
Fairly safe isolated change to baseline fitting code. Some changes to comment styles that maybe mess up doxygen but we can roll forward and fix that later if needed.

**Pull request merged**